File: Python/film.py
class Film:
    def __init__(self, id_val="kosong", nama="", genre="", harga=0, jual=0):
        self.__id = id_val
        self.__nama = nama
        self.__genre = genre
        self.__harga = harga
        self.__jual = jual

    # getter
    def getid(self):
        return self.__id
        
    def getnama(self):
        return self.__nama
        
    def getgenre(self):
        return self.__genre
        
    def getharga(self):
        return self.__harga
        
    def getjual(self):
        return self.__jual

    # setter
    def setid(self, text):
        self.__id = text
        
    def setnama(self, text):
        self.__nama = text
        
    def setgenre(self, text):
        self.__genre = text
        
    def setharga(self, angka):
        self.__harga = angka
        
    def setjual(self, angka):
        self.__jual = angka

File: java/Film.java
public class Film {
    private String id;
    private String nama;
    private String genre;
    private int harga;
    private int jual;

    // constructor I
    public Film() {
        this.id = "kosong";
    }

    // constructor II (ada parameter)
    public Film(String id, String nama, String genre, int harga, int jual) {
        this.id = id;
        this.nama = nama;
        this.genre = genre;
        this.harga = harga;
        this.jual = jual;
    }

    // getter
    public String getId() {
        return id;
    }

    public String getNama() {
        return nama;
    }

    public String getGenre() {
        return genre;
    }

    public int getHarga() {
        return harga;
    }

    public int getJual() {
        return jual;
    }

    // setter
    public void setId(String text) {
        id = text;
    }

    public void setNama(String text) {
        nama = text;
    }

    public void setGenre(String text) {
        genre = text;
    }

    public void setHarga(int angka) {
        harga = angka;
    }

    public void setJual(int angka) {
        jual = angka;
    }
}
